modules/ROOT/partials/configuration/line_height_formats.adoc

```diff
@@ -12,8 +12,8 @@ Default Value : `+'1 1.1 1.2 1.3 1.4 1.5 2'+`
 [source,js]
 ----
 tinymce.init({
-  selector: 'textarea', // change this value according to your HTML
-  toolbar: 'lineheight',
-  line_height_formats: '1 1.1 1.2 1.3 1.4 1.5 2'
+    selector: 'textarea', // change this value according to your HTML
+    toolbar: 'lineheight',
+    line_height_formats: '1 1.2 1.4 1.6 2'
```

Comment:
Looks like this has incorrect indentation now?

Reply:
I blame merge conflicts